README.md

```diff
@@ -145,3 +165,311 @@ Releases are aligned to Vega core [releases](https://github.com/vegaprotocol/veg
  - Run `make` followed by a full integration test run
  - Create a branch containing these changes, then a pull request into `develop`
  - Use the GitHub [New Release](https://github.com/vegaprotocol/vega-market-sim/releases/new) dialog to create a release. Tag it with the version of Vega we are matching up to (e.g. `v0.62.5`) to trigger the deployment process.
+
+
+## Windows Setup
+
+## 1. Install [WSL]
+To install Windows Subsystem for Linux (WSL) on Windows, follow these steps:
```

Comment:
I wouldn't give details here, it may be better to give a link to WSL on the MS website and leave it at that. 
```
1. Install WSL.
2. Install Ubuntu distribution inside WSL.
```

Reply:
Updated.